save-agent/src/nativeMain/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -112,7 +115,10 @@ class SaveAgent(private val config: AgentConfiguration,
                     state.value = AgentState.CLI_FAILED
                     return@coroutineScope
                 }
-                sendExecutionData(ExecutionData(emptyList()))
+                // todo: read data from files here
```

Comment:
This comment is not needed anymore, files are now read just above. You can change it to `parse test executions from files` though